server/server_media.go

```diff
@@ -13,6 +14,10 @@ import (
 	"github.com/status-im/status-go/signal"
 )
 
+var (
+	UseHTTP = false
```

Comment:
A global var can often solve problems rather easily because it allows code to skip pretty much all layers in a system, but it also causes spooky action at a distance that's hard to understand.

For example here:

```go
func main() {
	flag.Parse()

	mediaServer.UseHTTP = !*useMediaHTTPS

	srv := backendServer.NewServer()
	srv.Setup()

	err := srv.Listen(*address)
	...
	srv.RegisterMobileAPI()
	srv.Serve()
}
```

It's not a big deal, but is there a cleaner way to solve this problem?